offerzen/global/employers/employers@4.0.js

```diff
@@ -9,44 +9,44 @@ window.$loaded(function (window, document, $, undefined) {
         $parsleyLoaded(cb)
       }, 50)
     }
-  })();
-
-  window.$parsleyLoaded(function (window, document, parsley) {
-    function getRoleTypes(formData, formProperties) {
-      let fields = [
-        'role_type[In-office]',
-        'role_type[Fully-remote]',
-        'role_type[Hybrid]',
-      ]
-      let role_types = []
-
-      for (let key in formProperties) {
-        if (fields.includes(key) && formProperties[key] === 'on') {
-          if (key === 'role_type[Fully-remote]') {
-            role_types.push(
-              key.replace('role_type[', '').replace(']', '').replace('-', ' ')
-            )
-          } else {
-            role_types.push(key.replace('role_type[', '').replace(']', ''))
-          }
-          delete formProperties[key]
+  })()
+
+  function getRoleTypes(formData, formProperties) {
+    let fields = [
+      'role_type[In-office]',
+      'role_type[Fully-remote]',
+      'role_type[Hybrid]',
+    ]
+    let role_types = []
+
+    for (let key in formProperties) {
+      if (fields.includes(key) && formProperties[key] === 'on') {
+        if (key === 'role_type[Fully-remote]') {
+          role_types.push(
+            key.replace('role_type[', '').replace(']', '').replace('-', ' ')
+          )
+        } else {
+          role_types.push(key.replace('role_type[', '').replace(']', ''))
         }
+        delete formProperties[key]
       }
-
-      return role_types
     }
 
-    function tracking() {
-      dataLayer.push({
-        event: 'Company Lead Form Submitted ',
-        action: 'Lead Form Submitted',
-        label: 'Company Sign Up / Employer Landing Page',
-        category: 'Core',
-        source: 'Demand Sign Up',
-      })
-    }
+    return role_types
+  }
 
-    window.onSubmitCompanyLeadForm = function (token, e) {
+  function tracking() {
+    dataLayer.push({
+      event: 'Company Lead Form Submitted ',
+      action: 'Lead Form Submitted',
+      label: 'Company Sign Up / Employer Landing Page',
+      category: 'Core',
+      source: 'Demand Sign Up',
+    })
+  }
+
+  function onSubmitCompanyLeadForm(token) {
```

Comment:
There might be other references to this, e.g. recaptcha in WebFlow. Is it definitely not referenced elsewhere?